app/serializers/rest/dedup_notification_group_serializer.rb

```diff
@@ -1,7 +1,42 @@
 # frozen_string_literal: true
 
 class REST::DedupNotificationGroupSerializer < ActiveModel::Serializer
+  class PartialAccountSerializer < ActiveModel::Serializer
```

Comment:
Could we have a concern to avoid repeating the methods with the full account serializer, or would it cause unexpected complexity?

Reply:
I wonder about it, but I'm not sure it would make things more maintainable. What could make things more maintainable is have `PartialAccountSerializer` depend on `REST::AccountSerializer` but in a way that it only exports a bunch of attributes. Not sure how to do that, though.